src/main/kotlin/racing/RacingUI.kt

```diff
@@ -0,0 +1,11 @@
+package racing
+
+class RacingUI {
```

Comment:
특정 조건의 경우 `class` 대신 `object` 를 사용해 볼 수 있을것 같아요.
```suggestion
object RacingUI {
```